src/axom/quest/docs/sphinx/point_mesh_query_cpp.rst

```diff
@@ -71,15 +71,18 @@ a triangle surface mesh.  The second argument indicates the mesh is a watertight
 mesh, a manifold.  The signed distance from a point to a manifold is
 mathematically well-defined.  When the input is not a closed surface mesh, the
 mesh must span the entire computational mesh domain, dividing it into two regions.
-The third and fourth arguments are used to build the underlying BVH tree
-spatial index.  They indicate that BVH tree buckets will be split after 25 objects
-and that the BVH tree will contain at most 10 levels.  These are safe default 
-values and can be adjusted if application benchmarking shows a need.  Note that
-the second and subsequent arguments to the constructor correspond to
+The third argument is optional, and allows toggling the computation of signs in
```

Comment:
Can you list what the third and fourth arguments are here? It would be more clear imo.